tool/tsh/tsh.go

```diff
@@ -2832,6 +2833,17 @@ func serializeProfiles(profile *client.ProfileStatus, profiles []*client.Profile
 	return string(out), nil
 }
 
+func getStatusEnv() map[string]string {
+	env := map[string]string{}
+	envVars := []string{proxyEnvVar, clusterEnvVar, siteEnvVar, userEnvVar}
```

Comment:
I think these are the most relevant env vars. I could include TELEPORT_HOME and these env vars defined in tsh.go:

https://github.com/gravitational/teleport/blob/50fdea64136c83f446f6696088844c458b750873/tool/tsh/tsh.go#L375-L384

I'm not sure if those are relevant enough info to display though. We probably don't want to display every env var being picked up by tsh, right?